File: TaskManager/src/models/Comment.java
package models;

import java.time.LocalDateTime;

public class Comment {
    private int id;
    private String content;
    private String author;
    private LocalDateTime createdAt;

    public Comment(int id, String content, String author) {
        this.id = id;
        this.content = content;
        this.author = author;
        this.createdAt = LocalDateTime.now();
}
}

File: Lab 12/Lab12/Task_manager/Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

public enum Priority
{
    Low,
    Medium,
    High,
    Critical
}

public enum Status
{
   Open,
   Inprogress,
    UnderReview,
    Resolved,
    Closed
}

namespace Task_manager
{
    public class Comment
    {
       public int id;
       public string content;
       public string Author;
       public DateTime CreatedAt;

      public Comment(int id, string content, string Author, DateTime CreatedAt)
        {
            this.id = id;
            this.content = content;
            this.Author = Author;
            this.CreatedAt = CreatedAt;
        }


    }
}
